pxtlib/tutorial.ts

```diff
@@ -135,7 +124,7 @@ namespace pxt.tutorial {
                         m2 = "";
                         break;
                     case "tutorialValidationRules":
```

Comment:
why not remove the case here?

Reply:
Or remove "tutorialValidationRules" from getMetadataRegex?

Reply:
I suppose can remove them now, this was the first file I started cleaning things up from so I hadn't made up my mind to fully clean it out yet and I guess I didn't come back to do the last few lines since they didn't really change anything